src/middlewared/middlewared/plugins/system/debug.py

```diff
@@ -62,16 +63,21 @@ def progress_callback(percent, desc):
         except Exception as e:
             raise CallError(f'Failed to generate debug: {e!r}')
 
-    @accepts()
+    @accepts(
+        Dict(
+            'debug_options',
+            Dict('extra', additional_attrs=True)
```

Comment:
Can we define what the supported debug options are?

Reply:
Done